rust/worker/src/config.rs

```diff
@@ -70,10 +69,10 @@ impl RootConfig {
         }
         // Apply defaults - this seems to be the best way to do it.
         // https://github.com/SergioBenitez/Figment/issues/77#issuecomment-1642490298
-        f = f.join(Serialized::default(
-            "worker.num_indexing_threads",
-            num_cpus::get(),
-        ));
+        // f = f.join(Serialized::default(
```

Comment:
why are we leaving this commented out code? just to show defaults? Maybe we should use the defaulting for the number of threads in the dispatcher - num_worker_threads.